react/week3/src/components/TodoForm.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+
+function TodoForm({ addTodo }) {
+	const [ value, setValue ] = useState({

```

Comment:
I would name this `todo`. `value` is a very generic name.
```suggestion
	const [ todo, setTodo ] = useState({
```